exonum-java-binding-parent/exonum-java-binding-core/src/main/java/com/exonum/binding/storage/indices/ListSpliterator.java

```diff
@@ -32,7 +32,7 @@
  * associated with a given {@link com.exonum.binding.storage.database.Fork} is modified; not only
  * the source list.
  *
- * <p>This spliterator does not support specializations (e.g., {@link Spliterator.OfInt}).
+ * <p>This spliterator does not support specializations (e.g., {@link OfInt}).
```

Comment:
It is not clear in the text what `OfInt` is.

Reply:
Idea optimized it while merging with master :)